types/yawn-yaml/cjs/index.d.ts

```diff
@@ -0,0 +1,7 @@
+export default class YAWN {
+    constructor(content: string);
+    json: any;
+    yaml: string;
+    getRemark(path: string): any;
```

Comment:
Are you sure that you want to use so many `any`s?  For example, looking at the code, *I think* that `getRemark` returns an `undefined | string`...

Reply:
Good catch! I think it might actually just be `string`, as there's a `return ''` [in `getNodeRemark`](https://github.com/mohsen1/yawn-yaml/blob/master/src/index.js#L429-L455).

As for `json` that type comes directly [from `js-yaml#load`](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/types/js-yaml/index.d.ts#L10)